src/mixed-line-bar-chart/data-series.tsx

```diff
@@ -65,6 +65,13 @@ export default function DataSeries<T extends ChartDataTypes>({
     return calculateOffsetMaps(barData);
   }, [visibleSeries, stackedBars]);
 
+  const getStackedMinimum = (xValue: ChartDataTypes) => {
+    return stackedBarOffsetMaps?.minValues.get(getKeyValue(xValue)) ?? 0;
```

Comment:
When is this value undefined and why is it safe to assume 0 here?

Reply:
Logically, the min/max values are defined for each value so do not expect any to be undefined. However, I found how to make the overall code simpler - resolving the thread.